packages/core/src/mappings/LinearGradient.ts

```diff
@@ -9,71 +9,75 @@ export const SEED_DATA = {
   name: "Linear Gradient",
   tag: "LinearGradient",
   description: "Linear Gradient Component",
+  layout: { width: "100%", height: "100%" },
   props: {
-    color1: createColorProp({
-      label: "Color 1",
-    }),
-    color2: createColorProp({
-      label: "Color 2",
-    }),
-    color3: createColorProp({
-      label: "Color 3",
-    }),
-    startX: {
+    endY: {
       group: GROUPS.basic,
-      label: "Start X",
-      description: "Start position from Left",
+      label: "End Y",
+      description: "End position from Bottom",
       editable: true,
       required: false,
-      defaultValue: 0,
+      defaultValue: 100,
       min: 0,
       max: 100,
       step: 1,
       precision: 0,
       formType: FORM_TYPES.number,
       propType: PROP_TYPES.NUMBER,
     },
-    startY: {
+    endX: {
       group: GROUPS.basic,
-      label: "Start Y",
-      description: "Start position from Top",
+      label: "End X",
+      description: "End position from Right",
       editable: true,
       required: false,
-      defaultValue: 0,
+      defaultValue: 100,
       min: 0,
       max: 100,
       step: 1,
       precision: 0,
       formType: FORM_TYPES.number,
       propType: PROP_TYPES.NUMBER,
     },
-    endX: {
+    startY: {
       group: GROUPS.basic,
-      label: "End Y",
-      description: "End position from Right",
+      label: "Start Y",
+      description: "Start position from Top",
       editable: true,
       required: false,
-      defaultValue: 100,
+      defaultValue: 0,
       min: 0,
       max: 100,
       step: 1,
       precision: 0,
       formType: FORM_TYPES.number,
       propType: PROP_TYPES.NUMBER,
     },
-    endY: {
+    startX: {
       group: GROUPS.basic,
-      label: "End Y",
-      description: "End position from Bottom",
+      label: "Start X",
+      description: "Start position from Left",
       editable: true,
       required: false,
-      defaultValue: 100,
+      defaultValue: 0,
       min: 0,
       max: 100,
       step: 1,
       precision: 0,
       formType: FORM_TYPES.number,
       propType: PROP_TYPES.NUMBER,
     },
+    color3: createColorProp({
+      label: "Color 3",
+      defaultValue: undefined,
+    }),
+    color2: createColorProp({
+      label: "Color 2",
+      defaultValue: "rgba(251, 252, 253, 1)",
```

Comment:
Does this work ok inside the builder experience after you run the script? I'm pretty sure we typically use theme colors for these, e.g. `"primary"`, which will translate to `theme.colors.primary`